src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -51,5 +51,10 @@ public class SqlServerDataStoreConfiguration
         /// If set, the client id of the managed identity to use when connecting to SQL, if AuthenticationType == ManagedIdentity.
         /// </summary>
         public string ManagedIdentityClientId { get; set; }
+
+        /// <summary>
+        /// Specifies the statement timeout to set on ServerConnection.
+        /// </summary>
+        public int StatementTimeoutInSecs { get; set; } = 14400;
```

Comment:
Is this config accessible from the schema manager command-line tool and the schema manager Paas?

Reply:
This config is for oss command line tool. Once these changes gets merged and in a version upgrade PR for healthcare-shared-components nugets in health-paas, we will need to add this parameter [here](https://microsofthealth.visualstudio.com/Health/_git/health-paas?path=/src/SqlManagementApplication/SchemaManager.Paas/Commands/ApplyPaasCommand.cs&version=GBmaster&line=86&lineEnd=87&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)